contributed_definitions/NXmpes.nxdl.xml

```diff
@@ -228,7 +246,7 @@
                 <field name="sample_bias" type="NX_FLOAT" recommended="true" units="NX_CURRENT"/>
             </group>
         </group>
-        <group type="NXprocess">
+        <group type="NXprocess" recommended="true">
```

Comment:
I set this to recommended, because all subgroups are optional, so this just required an empty group. Or was there a specific reason to always require an NXprocess group here?

Reply:
I agree with making NXprocess optional here.

Reply:
agrred

Reply:
Agreed